src/main/java/com/commercetools/sync/inventories/InventoryService.java

```diff
@@ -0,0 +1,69 @@
+package com.commercetools.sync.inventories;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * Interface contains methods to perform sphere queries/commands related to Inventory topic.
+ * Listed operations are used in {@link InventorySync}.
```

Comment:
this comment might be harmful. As a developer of implementation hidden behind interface you should not
care about where it is used. What if in the future this service is used elsewhere?